book-a-reading-room-visit.test/EmailTemplate/Text/BookingConfirmation.txt

```diff
@@ -17,7 +17,6 @@
     Visit type:                            {VisitType}
     Visit date:                            {VisitStartDateDisplay}
     Reading room:                          {ReadingRoom}
-    Seat number:                           {SeatNumber}
     
     Based on availability, we may need to change your seat on the date of your visit.</span>
```

Comment:
I wonder should we remove "Based on availability, we may need to change your seat on the date of your visit." too.